README.md

```diff
@@ -2,21 +2,20 @@
 # Open Energy Family - Organisation
 
 This repository is created for discussions and decisions regarding the Open Energy Family.
-It is a collection of various information and tools that help to "set energy data free".
-The Open Energy Family is currently and on the long-run under development.
+It is a collection of various tools and information and that help working with energy related data. It is a collaborative community effort, everything is openly developed. 
```

Comment:
Suggest to write: The Open Energy Family (OEF) is a collection of various tools and information and that provide support in working with energy related data. Te OEF is a collaborative community effort and everything is openly developed.